src/util.h

```diff
@@ -31,7 +31,6 @@ int wifi_get_freq24(int channel);
 int wifi_get_freq5(int channel);
 
 gchar* systemd_escape(char* string);
-gchar* netplan_get_id_from_nm_filename(const char* filename, const char* ssid);
```

Comment:
This is intentionally moved to the new `nm-keyfile.c/h` module. As it derives the netplan-ID from the keyfile filename and is specific to NM keyfiles.